src/authorization/WebAclAuthorizer.ts

```diff
@@ -24,17 +25,24 @@ import { Authorizer } from './Authorizer';
 export class WebAclAuthorizer extends Authorizer {
   protected readonly logger = getLoggerFor(this);
 
-  private readonly aclManager: AclManager;
+  private readonly aclManager: AuxiliaryManager;
```

Comment:
Rename var too?

Reply:
I actually kept this name on purpose to indicate that we specifically have an auxiliary manager for acl files here.